apps/web/src/components/(dashboard)/layout/verify-email-banner.tsx

```diff
@@ -0,0 +1,44 @@
+'use client';
+
+import { useState } from 'react';
+
+import { trpc } from '@documenso/trpc/react';
+import { Button } from '@documenso/ui/primitives/button';
+
+export const VerifyEmailBanner = ({ user }) => {
+  const [isButtonDisabled, setIsButtonDisabled] = useState(false);
+  const { mutateAsync: sendConfirmationEmail } = trpc.profile.sendConfirmationEmail.useMutation();
```

Comment:
Can use the `isLoading` option from the mutation here instead of using out own state.

Reply:
I want to disable the button for 20 seconds before the user can press again. It's a poor man's prevention from having users spam the button, unless we have a proper mechanism on the backend.

Is that doable with the `isLoading` option?